api/integrations/perforce/git_p4.py

```diff
@@ -3046,7 +3046,7 @@ def run(self, args):
         return self.rebase()
 
     def rebase(self):
-        if os.system("git update-index --refresh") != 0:
+        if system('git update-index --refresh') != 0:
```

Comment:
this file is 3rd-party code so these changes will realistically get thrown away, but I guess it's fine to fix it.